R/readUNFCCC_NDC.R

```diff
@@ -30,61 +31,160 @@ readUNFCCC_NDC <- function(subtype, subset) {
+        dplyr::mutate(
+          `Unconditional Relative` = NA_character_,
+          `Conditional Relative` = NA_character_,
+          `Unconditional Absolute` = NA_character_,
+          `Conditional Absolute` = NA_character_,
+          `Type` = NA_character_
+        ) %>%
+        # --- fill values by different types ---
+        dplyr::mutate(
+          # base year unconditional relative
+          `Unconditional Relative` = dplyr::if_else(
+            .data$`Type of NDC` == "Base year" & .data$Conditionality == "Unconditional",
+            as.character(.data$`reduction min (%)...21`),
+            .data$`Unconditional Relative`
+          ),
+          # base year conditional relative
+          `Conditional Relative` = dplyr::if_else(
+            .data$`Type of NDC` == "Base year" & .data$Conditionality == "Conditional",
+            as.character(.data$`reduction max (%)...22`),
+            .data$`Conditional Relative`
+          ),
+          # unconditional absolute emission targets
+          `Unconditional Absolute` = dplyr::if_else(
+            .data$`Type of NDC` == "Specific" & .data$Conditionality == "Unconditional",
+            as.character(.data$`emission level min (Mt CO2eq)...16`),
+            .data$`Unconditional Absolute`
+          ),
+          # conditional absolute emission targets
+          `Conditional Absolute` = dplyr::if_else(
+            .data$`Type of NDC` == "Specific" & .data$Conditionality == "Conditional",
+            as.character(.data$`emission level max (Mt CO2eq)...17`),
+            .data$`Conditional Absolute`
+          ),
+          # BAU unconditional
+          `Unconditional Relative` = dplyr::case_when(
+            .data$`Type of NDC` == "BAU" & .data$Conditionality == "Unconditional" 
+            & !is.na(.data$`reduction min (%)...18`) ~
+              as.character(.data$`reduction min (%)...18`),
+            TRUE ~ .data$`Unconditional Relative`
+          ),
+          `Unconditional Absolute` = dplyr::case_when(
+            .data$`Type of NDC` == "BAU" & .data$Conditionality == "Unconditional" 
+            & is.na(.data$`reduction min (%)...18`) ~
+              as.character(.data$`emission level min (Mt CO2eq)...16`),
+            TRUE ~ .data$`Unconditional Absolute`
+          ),
+          # BAU conditional
+          `Conditional Relative` = dplyr::case_when(
+            .data$`Type of NDC` == "BAU" & .data$Conditionality == "Conditional" 
+            & !is.na(.data$`reduction min (%)...18`) ~
+              as.character(.data$`reduction min (%)...18`),
+            TRUE ~ .data$`Conditional Relative`
+          ),
+          `Conditional Absolute` = dplyr::case_when(
+            .data$`Type of NDC` == "BAU" & .data$Conditionality == "Conditional" 
+            & is.na(.data$`reduction min (%)...18`) ~
+              as.character(.data$`emission level min (Mt CO2eq)...16`),
+            TRUE ~ .data$`Conditional Absolute`
+          ),
```

Comment:
Same here for BAU target type: do we take the BAU values that PBL provides into account for calculating the 2035 target emissions? Or do we make our own assumptions later?

Reply:
I agree. Might be good to know that.